### ISSUE TITLE
Allow markup in other places

### ISSUE THREAD

**Issue by yawetag** [buttonmen-dev/buttonmen#1518]
Markup in the forums is great. Can we extend it to create_game and even the chat for games?

**Comment by yawetag** [comment]
Also, allow new lines in create_game

**Comment by cgolubi1** [comment]
Hmm?  You can already use the forum markup within game chat.  What's in this issue which is actually unimplemented?

**Comment by dwvanstone** [comment]
I believe forum markup works in game chat. I verified it does not work in game description text when creating a game.

**Comment by irilyth** [comment]
Huh, evidently. I never realized this; there's nothing that indicates that it works there. Is the "Available markup" box from the forums too unwieldy to put on the game pages next to the chat box?

**Comment by dwvanstone** [comment]
Looking at the code, I think but forum markup is accidentally being applied to the game chat. 

I am looking into this area of the code (for bug #1684) and may end up turning off this "feature" in the game chat.

Do we *want* to have the game chat include "forum markups"?

**Comment by dwvanstone** [comment]
By the way, it is also "accidentally" applied to the comment in one's profile text.

**Comment by cgolubi1** [comment]
I don't think this is an accident.  I think we want to use the same markup in all text boxes.  Why turn it off?  Let me try to turn up the issues in which we originally discussed this for some evidence that it was intentional.

**Comment by cgolubi1** [comment]
So, #1079 is where we turned on markup parsing in the comment field of the profile.

**Comment by cgolubi1** [comment]
Also, there's an open issue, #1026, for describing the markup legend in game chat.

**Comment by dwvanstone** [comment]
I stand corrected! Okay, I'll make sure the function called makes it clear that the markup is being applied. (There are some cases where the markup isn't desired, such as in a forum title.)

**Comment by cgolubi1** [comment]
Sounds good, thanks.
